### PR TITLE
Fixing the Coroutine example

### PR DESCRIPTION
**Pull request by babyboucher**

### CODE REVIEW

**Comment by Thundermaker300** [comment]
documentation commits 😳

**Comment by babyboucher** [comment]
I tried to copy and paste it and got annoyed it was wrong

File: README.md [hunk 1]
```diff
@@ -111,8 +113,8 @@ public IEnumerator<float> MyCoroutine()
 {
     for(;;) //repeat the following infinitely
     {
-        Plugin.Info("Hey I'm a infinite loop!"); //Call Plugin.Info to print a line to the game console/server logs
-        yield return Timing.WaitForSecond(5f); //Tells the coroutine to wait 5 seconds before continuing, since this is at the end of the loop, it effectively stalls the loop from repeating for 5seconds.
```
Comment (Michal78900) [inline README.md:115]: `5 seconds.` instead `5seconds.`

File: README.md [hunk 2]
```diff
@@ -111,8 +113,8 @@ public IEnumerator<float> MyCoroutine()
...
+        Log.Info("Hey I'm a infinite loop!"); //Call Log.Info to print a line to the game console/server logs
+        yield return Timing.WaitForSeconds(5f); //Tells the coroutine to wait 5 seconds before continuing, since this is at the end of the loop, it effectively stalls the loop from repeating for 5seconds.
```
Comment (Michal78900) [inline README.md:117]: Same as above

File: README.md [hunk 3]
```diff
@@ -111,8 +113,8 @@ public IEnumerator<float> MyCoroutine()
 {
     for(;;) //repeat the following infinitely
```
Comment (Michal78900) [inline README.md:114]: you can add space bettween `for` and `(`

**Review by joker-119 — APPROVED** [review]

**Review by iopietro — CHANGES_REQUESTED** [review]
Fix what @Michal78900 said and then it's fine.

**Comment by babyboucher** [comment]
Done

**Review by Michal78900 — APPROVED** [review]

**Review by iopietro — APPROVED** [review]

**Pull request merged**